problems/arraysIntro.js

```diff
@@ -4,57 +4,83 @@
  * @param {string} fruit
  * @returns {string[]}
  */
-function addFruitToEnd() {}
+function addFruitToEnd(fruits,fruit) {
+   fruits.push(fruit);
+   return fruits;
+
+}
 
 /**
  * Adds a fruit to the front of an array of fruits and returns the array.
  * @param {string[]} fruits
  * @param {string} fruit
  * @returns {string[]}
  */
-function addFruitToFront() {}
+function addFruitToFront(fruits,fruit){
+fruits.unshift(fruit)
```

Comment:
Make sure to tab everything in the functions code block to maintain proper formatting and to keep everything easily readable.